index.bs

```diff
@@ -1017,8 +1017,8 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 			<code>{ name: "clipboard-read", allowWithoutGesture: true }</code>
 			permission.
 
-		1. Let |hasGesture| be true if the current script is not running as a result of
-			direct user action, false otherwise.
+		1. Let |hasGesture| be true if the [=relevant global object=] of [=this=] has
+		    [=transient activation=], false otherwise.
 
 		1. If |readWithoutGesture| is <a>granted</a>, then return true.
```

Comment:
While at it:

```suggestion
		1. If |readWithoutGesture| is <a>granted</a>, then return true.

		1. Let |hasGesture| be true if the [=relevant global object=] of [=this=] has
		    [=transient activation=], false otherwise.
```